common/source/docs/common-qiotek-zealoth7.rst

```diff
@@ -0,0 +1,541 @@
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>5</td>
+   <td>BUZZER-</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>6</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+Relay/ADC2 port
+---------------
+
+.. raw:: html
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>PIN</th>
+   <th>SIGNAL</th>
+   <th>VOLT</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>ADC2</td>
+   <td>+6V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>Relay1</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>Relay2</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>Relay3</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>5</td>
+   <td>Relay4</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>6</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+USART3(GPS1/I2C1), UART4(GPS2/I2C2) ports
+-----------------------------------------
+
+.. raw:: html
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>PIN</th>
+   <th>SIGNAL</th>
+   <th>VOLT</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>TX</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>RX</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>SCL</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>5</td>
+   <td>SDA</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>6</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+Power1, Power2 ports
+--------------------
+
+.. raw:: html
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>PIN</th>
+   <th>SIGNAL</th>
+   <th>VOLT</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>CURRENT</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>VOLTAGE</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>5</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   <tr>
+   <td>6</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+
+.. image:: ../../../images/qiotek-zealot-power-input.jpg
+
+
+VBAT Input
+----------
+
+30V maximum input. A 5V, 1.5A regulator drops this input to the +5V used and supplied by the autopilot. Do not connect power to this input if Power Module 2 input is used as they share the BATT2 voltage monitoring circuitry. This connector is a JST-EH type.
+
+SERVO VCC Rail Input
+--------------------
+
+This provides a means to power servos attached to the output connector block by an external BEC or power source. Maximum power rating is 36V @ 20A.
+
+RC Input
+--------
+
+All compatible RC protocols can be decoded by attaching the Receiver's output to the SBUS input pin next to the Servo/Output VCC input connector. Note that some protocols such as CRSF or, FPort including telemetry, require connection to, and setup of, one of the UARTs instead of this pin.
```

Comment:
```suggestion
All compatible RC protocols can be decoded by attaching the Receiver's output to the SBUS input pin next to the Servo/Output VCC input connector. Note that some protocols such as CRSF or FPort including telemetry, require connection to, and setup of, one of the UARTs instead of this pin.
```

Reply:
thanks